### PR TITLE
Priority feature

### LINKED ISSUE #85: Implement Priority Feature
User Story:  As a/an expert user, I can add priority tag to clients, so that I can prioritise them for e.g. meetings

As a visual and artistic person, I can tag my contacts using colour and categorize them, so that I can make my contact list more visually stimulating.

Example usage: 
`add /n Meme [/l 4]`
This would create a person whose name is Meme and have a priority of 4.

`add /n Meme`
This would create a person whose name is Meme and have an undefined priority

### PR DESCRIPTION
**Pull request by Joven-Heng**
close #85 

Implemented Priority Feature with UI

### CODE REVIEW

File: src/main/java/seedu/address/ui/PersonCard.java [hunk 1]
```diff
@@ -76,6 +91,40 @@ public PersonCard(Person person, int displayedIndex) {
...
+        case "high":
+            priorityShape.setBorder(new Border(new BorderStroke(ColorPicker.RED_BORDER,
+                    BorderStrokeStyle.SOLID, CornerRadii.EMPTY, new BorderWidths(BORDER_SIZE))));
+            priorityShape.setBackground(new Background(new BackgroundFill(ColorPicker.RED, CornerRadii.EMPTY,
+                    Insets.EMPTY)));
+            break;
+        case "undefined":
+            priorityShape.setBorder(new Border(new BorderStroke(ColorPicker.WHITE_BORDER,
+                    BorderStrokeStyle.SOLID, CornerRadii.EMPTY, new BorderWidths(BORDER_SIZE))));
+            priorityShape.setBackground(new Background(new BackgroundFill(Color.WHITE, CornerRadii.EMPTY,
```
Comment (ijavierja) [inline src/main/java/seedu/address/ui/PersonCard.java:121]: Is there a difference between ColorPicker.WHITE_BORDER and Color.WHITE?

**Review by ijavierja — COMMENTED** [review]
If the abnormality highlighted in the comment was on purpose, then LGTM

File: src/main/java/seedu/address/ui/PersonCard.java (hunk 1 above)
Comment (Joven-Heng) [reply src/main/java/seedu/address/ui/PersonCard.java:121]: Yes, there is, white border is slightly darker

**Review by WeiJie96 — CHANGES_REQUESTED** [review]
Just a small comment on the javadoc

File: src/main/java/seedu/address/ui/ColorPicker.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package seedu.address.ui;
+
+import javafx.scene.paint.Color;
+
+/**
+ * Class for custom colors used for AB3, especially for the priorityShape
```
[the remark this answers is not in the document]
Comment (Joven-Heng) [reply src/main/java/seedu/address/ui/ColorPicker.java:6]: wa shag forgot my bad.

**Comment by Joven-Heng** [comment]
I would feel theres a certain level of danger to this because there isnt tests written for these yet, but I assume its just the first iteration of it so yeah LGTM.

**Review by WeiJie96 — APPROVED** [review]
LGTM

**Pull request merged**